integration_tests/src/main/python/fastparquet_compatibility_test.py

```diff
@@ -94,6 +94,13 @@ def read_with_fastparquet_or_plugin(spark):
     return read_with_fastparquet_or_plugin
 
 
+def is_timezone_utc():
```

Comment:
There are some APIs that do similar things to this already.

https://github.com/NVIDIA/spark-rapids/blob/b3f1e4744bcee969505cf079a79cd1c308527a97/integration_tests/src/main/python/conftest.py#L83-L87

I think it is better if we use these APIs and then we can improve them if we have a better way to deal with UTC time zone checks.

Reply:
Agreed. Fixed.